<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browserSelect</class>
 <widget class="QDialog" name="browserSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="btnBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>295</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbtnDefault">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>默认浏览器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnChrome">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>谷歌浏览器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnFirefox">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>火狐浏览器</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnEdge">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edge浏览器</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>请选择浏览器：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>browserSelect</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>browserSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
